portafly/src/components/Loading.tsx

```diff
@@ -7,12 +7,17 @@ import {
   Title,
   Spinner
 } from '@patternfly/react-core'
+import { useTranslation } from 'i18n/useTranslation'
 
-export const Loading: React.FunctionComponent = () => (
-  <PageSection aria-label="Loading Content Container">
-    <EmptyState variant={EmptyStateVariant.full}>
-      <EmptyStateIcon variant="container" component={Spinner} />
-      <Title headingLevel="h2" size="lg">Loading</Title>
-    </EmptyState>
-  </PageSection>
-)
+export const Loading: React.FunctionComponent = () => {
+  const { t } = useTranslation('shared')
+  return (
+  // FIXME: missing translation string
+    <PageSection aria-label={t('loading_aria_label')}>
```

Comment:
Instead of leaving a nonexistent string here, why not creating the string in the lang file, using the current value, and add the TODO/FIXME comment ?
In shared.yml, we can add something like
```
aria_labels:
  loading: Loading Content Container
```